src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -342,13 +342,13 @@ private function createVariants(ProductInterface $product, array $options)
 
     /**
      * @param ProductVariantInterface $productVariant
-     * @param ChannelInterface $channel
+     * @param string $channelCode
      */
-    private function createChannelPricings(ProductVariantInterface $productVariant, ChannelInterface $channel)
+    private function createChannelPricings(ProductVariantInterface $productVariant, $channelCode)
     {
         /** @var ChannelPricingInterface $channelPricing */
         $channelPricing = $this->channelPricingFactory->createNew();
-        $channelPricing->setChannel($channel);
+        $channelPricing->setChannel($channelCode);
```

Comment:
It can be done later, but maybe we should rename `setChannel` to `setChannelCode`?